service/api/service.go

```diff
@@ -119,6 +133,11 @@ func (s *serviceImpl) ApiV1WorkflowStartPost(
 				workflowOptions.Memo[da.GetKey()] = da.GetValue()
 			}
 		}
+		if requestId != nil {
+			workflowOptions.Memo[service.WorkflowAlreadyStartedRequestId] = iwfidl.EncodedObject{
+				Data: requestId,
+			}
```

Comment:
@longquanzheng when saving `WorkflowAlreadyStartedRequestId` to memo, should it be an EncodedObject or a simple string?